src/test/resources/example.yml

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: Vanilla CloudFoundry example
+location:
+    cloudfoundry:
+      user: user@mail.com
+      password: secret
+      org: secret_organization
+      endpoint: https://api.run.secret.io
+      space: development
+      address: run.secret.io
+      memory: 1024
+      disk_quota: 10
+      instances: 1
```

Comment:
I think `memory`, `disk_quota` and `instances` should be configured from the entity too. WDYT?